rules/prebuilt_frameworks_importer.bzl

```diff
@@ -0,0 +1,177 @@
+# The default verbose level when running this rules
```

Comment:
these are repository rules, right? should they be namespaced into a different package to make that clear?

Reply:
Actually, are these only meant for tests? If so, can we move it into the tests package?

Reply:
Yes, this are repository rules. I can namespace them. Do you mean that instead of placing them in `rules/` to place them inside a subdirectory with a different name? (any name proposals?)

Not only ment for tests, I find this a super great way of integrating 3rd party in bazel, because you also get them cached :). I was thinking of this as rules to use by everybody

Reply:
maybe inside `repository_rules` ? open to @ob's suggestions here

Reply:
`repository_rules` sgtm